crates/turborepo-lib/src/cli/mod.rs

```diff
@@ -1214,6 +1214,18 @@ pub async fn run(
             generate::run(tag, command, &args, child_event)?;
             Ok(0)
         }
+        Command::Info => {
+            CommandEventBuilder::new("info")
+                .with_parent(&root_telemetry)
+                .track_call();
+            let base = CommandBase::new(cli_args.clone(), repo_root, version, color_config);
+
+            match info::run(base).await {
+                Ok(()) => {}
+                Err(e) => println!("Command failed. Please file a GitHub Issue.\n{}", e),
```

Comment:
I'd just panic here. The panic handler is pretty clear on filing a GitHub issue and will include a backtrace

Reply:
TIL